proyecto-angular-pasante5/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
El unit test no deberia estar en el primer pull request.

Reply:
Disculpe es el archivo que se crea por defecto con ng g c el unit test que añadi es  it('eliminar lugar de lista de lugares', () => {
    expect(component).toBeTruthy();
    component.lugares=[{nombreLugar:"lugar 1",descripcion:"descripcion",coordenadas:['-18.9120921','-17.89328392832']}];
    component.eliminarElemento(0);
    expect(component.lugares).toEqual([]);
  });

Reply:
Añadi el test eliminar lugar de lista de lugares en la rama unit_test